src/gluonts/nursery/duru/README.md

```diff
@@ -0,0 +1,64 @@
+# Project Duru: Hierarchical VAEs and diffusions for forecasting and anomaly detection
+
+This repository contains the source code of 'Project Duru: Hierarchical VAEs (HVAEs) and diffusions for forecasting and anomaly detection'.
+At present, the repository contains:
+
+* Two working implementations of a conditional VDVAE [1] model for time-series forecasting based on a fully-connected and convolutional architecture.
+* Implementations of a simple "two Gaussian" dataset, a sine wave dataset, and all gluonts datasets within an appropriate training pipeline
+* Logging of plots and metrics to supervise the optimization process and analyse results, e.g. ELBO, distortion, rate, NaN counts, reconstruction plots, prediction plots, a test routine based on the gluonts `Evaluator` class, ...
+
+All hyperparameters are extensively explained in the help strings of the command line arguments in `hyperparams.py`.
+This source code partly builds on top of the [official implementation of VDVAE](https://github.com/openai/vdvae) and a previously developed implementation of VDVAE by the author of this repository.
+
+The source code is in a preliminary state, and it will be refined as the project matures.
+
+[1] Child, R., 2020. Very deep vaes generalize autoregressive models and can outperform them on images. arXiv preprint arXiv:2011.10650.
+
+## Installation
+
+Please set up a virtual environment installing the following key dependencies: 
+
+```shell
+pip install torch torchvision
```

Comment:
You could add a requirements file for this.

Reply:
Done.